operations/host_provision.go

```diff
@@ -98,17 +101,13 @@ func makeHostProvisioningScriptFile(workingDir string, content string) (string,
 	// slashes ('/') instead as the path separator.
 	scriptPath = util.ConsistentFilepath(scriptPath)
 	if err = ioutil.WriteFile(scriptPath, []byte(content), 0700); err != nil {
-		return "", errors.Wrapf(err, "writing script to file '%s'", scriptPath)
+		return "", errors.Wrapf(err, "writing script file '%s'", scriptPath)
 	}
 	return scriptPath, nil
 }
 
-func hostProvisioningCommand(shellPath, scriptPath string) *jasper.Command {
-	return jasper.NewCommand().AppendArgs(shellPath, "-l", scriptPath)
-}
-
-func runHostProvisioningCommand(ctx context.Context, cmd *jasper.Command) error {
-	cmd.SetOutputWriter(utility.NopWriteCloser(os.Stdout)).SetErrorWriter(utility.NopWriteCloser(os.Stderr))
```

Comment:
where does the output go now?

Reply:
I changed it to produce output for non-Windows hosts. On Windows, we get no output.

Reply:
would it get around the limits if we set the output writer to write a file?

Reply:
and if it did, would we need to remove `set -o verbose` on Windows?

Reply:
Maybe, but I don't think it would be that useful. The user data logs are mostly useful for debugging purposes.